src/ceph-volume/ceph_volume/tests/test_terminal.py

```diff
@@ -1,4 +1,10 @@
+# -*- mode:python; tab-width:4; indent-tabs-mode:t; coding:utf-8 -*-
+
+import codecs
+import io
+import mock
```

Comment:
this is not going to work :( ceph-volume (and ceph) do not depend on Mock, and although it is being added to tox.ini as a dependency it will break package workflows that require declaring dependencies for running tests.

Instead, we can use py.test's own mock framework (available as a test fixture) called `monkeypatch`

Reply:
thanks. TIL.